estuary/api/v1.py

```diff
@@ -92,18 +92,18 @@ def get_resource_story(resource, uid):
         raise NotFound('This item does not exist')
 
     forward_query = create_story_query(item, item.id, limit=True)
-    backward_query = create_story_query(item, item.id, reverse=True, limit=True)
+    backward_query = create_story_query(item, item.id, forward=True, limit=True)
```

Comment:
I think we can keep this `reverse` since the query is actually being formed in the reverse direction of the story when the flag is true